it/controllers/APIControllerISpec.scala

```diff
@@ -53,7 +53,6 @@ class APIControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock wit
         |       {
         |          "penaltyCategory": "LPP2",
         |          "penaltyStatus": "A",
-        |          "penaltyAmountAccruing": 123.45,
```

Comment:
Accruing penalties have the penaltyAmountAccruing field

Reply:
Agree, but there was 2 of them...line 67 shows another one

Reply:
Ah yes! Whoops!